ister.py

```diff
@@ -408,43 +411,62 @@ def get_current_format():
         frmt = format_file.read().strip()
     return frmt
 
+def add_default_dnf_config(args):
+    """Supply a default dnf config if one does not exist
+    """
+    os.makedirs(os.path.basename(args.dnf_config), exist_ok=True)
+    with open(args.dnf_config, 'w') as file:
+        file.write("[clear]\nname=clear\nbaseurl=https://download.clearlinux.org/current/x86_64/os/\nenabled=1\n\n[main]\nlogfile=/var/log/dnf.log\n")
 
 def copy_os(args, template, target_dir):
     """Wrapper for running install command
     """
-    LOG.info("Starting swupd. May take several minutes")
-    if not args.format:
-        args.format = get_current_format()
-    add_bundles(template, target_dir)
-    swupd_command = "swupd verify --install --path={0} " \
-                    "--manifest={1}".format(target_dir, template["Version"])
+    package_manager = template["PackageManager"]
+    LOG.info("Starting {0}. May take several minutes".format(package_manager))
+
+    if package_manager == SWUPD:
+        if not args.format:
+            args.format = get_current_format()
+        if args.fast_install:
+            args.statedir = "{0}/tmp/swupd".format(target_dir)
+        add_bundles(template, target_dir)
+        cmd = "swupd verify --install"
+        cmd += " --path={0}".format(target_dir)
+        cmd += " --manifest={0}".format(template["Version"])
+        cmd += " --contenturl={0}".format(args.contenturl)
+        cmd += " --versionurl={0}".format(args.versionurl)
+        cmd += " --format={0}".format(args.format)
+        cmd += " --statedir={0}".format(args.statedir)
+        if args.cert_file:
+            cmd += " -C {0}".format(args.cert_file)
+    else:
```

Comment:
I'm inclined to match specificly and error.

Reply:
Match specifically?  Maybe we could default the cert file to the one on the system for swupd?

Reply:
Ah sorry I mean if SWUPD elif DNF else raise error

Reply:
Ah, yessir!  I'll add this in.